Issue
  { issueClosedAt = Just 2014 (-03) (-18) 06 : 00 : 11 UTC
  , issueUpdatedAt = 2014 (-03) (-18) 06 : 02 : 39 UTC
  , issueEventsUrl =
      "https://api.github.com/repos/rethinkdb/rethinkdb/issues/1706/events"
  , issueHtmlUrl =
      Just "https://github.com/rethinkdb/rethinkdb/issues/1706"
  , issueClosedBy = Nothing
  , issueLabels =
      [ IssueLabel
          { labelColor = "e102d8"
          , labelUrl =
              "https://api.github.com/repos/rethinkdb/rethinkdb/labels/tp:bug"
          , labelName = "tp:bug"
          }
      ]
  , issueNumber = 1706
  , issueAssignee = Nothing
  , issueUser =
      SimpleUser
        { simpleUserId = Id 505365
        , simpleUserLogin = N "danielmewes"
        , simpleUserAvatarUrl =
            "https://avatars.githubusercontent.com/u/505365?v=3"
        , simpleUserUrl = "https://api.github.com/users/danielmewes"
        , simpleUserType = OwnerUser
        }
  , issueTitle =
      "Illegal to destroy fifo_enforcer_sink_t while outstanding exit_write_t objects exist"
  , issuePullRequest = Nothing
  , issueUrl =
      "https://api.github.com/repos/rethinkdb/rethinkdb/issues/1706"
  , issueCreatedAt = 2013 (-11) (-26) 00 : 57 : 09 UTC
  , issueBody =
      Just
        "While experimenting with a 64 node cluster (on EC2), I ran into heartbeat timeouts. Then one of the nodes crashed:\n\n```\nerror: Error in ./src/concurrency/fifo_enforcer.hpp at line 222:\nerror: illegal to destroy fifo_enforcer_sink_t while outstanding exit_write_t objects exist\nerror: Backtrace:\nerror: Tue Nov 26 00:54:50 2013\n\n       1: <unknown function> at 0x9fc3f5 ()\n       2: <unknown function> at 0x9fcba3 ()\n       3: <unknown function> at 0x9fbe54 ()\n       4: <unknown function> at 0xa06bb9 ()\n       5: <unknown function> at 0x8426d7 ()\n       6: <unknown function> at 0x842472 ()\n       7: <unknown function> at 0x9b6847 ()\n       8: <unknown function> at 0x9bbe92 ()\n       9: <unknown function> at 0x42cecb ()\nerror: Exiting.\n```\n\nWill try to get the symbols later.\n"
  , issueState = "closed"
  , issueId = Id 23286766
  , issueComments = 11
  , issueMilestone =
      Just
        Milestone
          { milestoneCreator =
              SimpleUser
                { simpleUserId = Id 706854
                , simpleUserLogin = N "AtnNn"
                , simpleUserAvatarUrl =
                    "https://avatars.githubusercontent.com/u/706854?v=3"
                , simpleUserUrl = "https://api.github.com/users/AtnNn"
                , simpleUserType = OwnerUser
                }
          , milestoneDueOn = Nothing
          , milestoneOpenIssues = 1
          , milestoneNumber = 17
          , milestoneClosedIssues = 595
          , milestoneDescription =
              Just
                "The scope of this issue is covered by another issue. The closing comment should link to the other issue."
          , milestoneTitle = "duplicate"
          , milestoneUrl =
              "https://api.github.com/repos/rethinkdb/rethinkdb/milestones/17"
          , milestoneCreatedAt = 2013 (-03) (-29) 20 : 23 : 12 UTC
          , milestoneState = "closed"
          }
  }